examples/evaluation/lora-evaluation.ipynb

```diff
@@ -0,0 +1,4712 @@
+       " 'transformers_version': '4.45.1',\n",
+       " 'upper_git_hash': None,\n",
+       " 'tokenizer_pad_token': ['[PAD]', '0'],\n",
+       " 'tokenizer_eos_token': [None, 'None'],\n",
+       " 'tokenizer_bos_token': [None, 'None'],\n",
+       " 'eot_token_id': None,\n",
+       " 'max_length': 512}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# use the path of your checkpoint here\n",
+    "lm_eval.simple_evaluate(model = 'hf',\n",
```

Comment:
Instead of showing the full output, let's only show the relevant part. This should work:

```python
output = lm_eval.simple_evaluate(...)
output["results"]
```

Same below.